packages/realm/bindgen/js_opt_in_spec.yml

```diff
@@ -51,6 +51,7 @@ records:
       - initialization_function
       - should_compact_on_launch_function
       - automatically_handle_backlinks_in_migrations
+      - exclude_from_icloud_backup
```

Comment:
These values refer to fields of the `RealmConfig` defined by Realm Core: https://github.com/realm/realm-core/blob/43a1aac3dc41edf80a5be093a8a65e839b0ad225/src/realm/object-store/shared_realm.hpp#L94-L194

If it's important for this behavior to be configurable, an alternative that wouldn't require changes to Core would be pass this as a bool argument to `after_realm_open` 🤔 Or rename `after_realm_open` to `exclude_from_icloud_backup` and call it conditionally in `Realm.ts`.

Reply:
Oh at first I thought the only way to add parameters to the config was adding on realm-core, but I see now we can scope it just to realm-js, so I'll go with the parameter option.